docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
+
+{{"demo": "CustomMaterialIcons.js", "defaultCodeOpen": true}}
+
+Or you can create your own components:
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Adding borders
+
+{{"demo": "BorderedTreeView.js", "defaultCodeOpen": false}}
+
+### Customizing collapse animations
```

Comment:
```suggestion
### Expand/collapse animation
```

I guess we can ditch the "customizing" verb here, given it's already in the Customization section?
Shorter TOC! 😬

Reply:
What do you think about 'Toggle animations'? Having 'Expand/collapse' like that in the title feels weird to me 🤔